### ISSUE TITLE
OSError: No such file or directory for scan_csv

### ISSUE THREAD

**Issue by tonyabracadabra** [pola-rs/polars#2960]
```
>>> df = pl.scan_csv('~/Documents/adult.csv')
Traceback (most recent call last):
... envs/analytics/lib/python3.9/site-packages/polars/internals/lazy_frame.py", line 133, in scan_csv
    self._ldf = PyLazyFrame.new_from_csv(
OSError: No such file or directory (os error 2)
>>> df = pl.read_csv('~/Documents/adult.csv')
```

Same operation for `pl.read_csv` works for me

I've tried several files and the results look the same

**Comment by ritchie46** [comment]
This will help on that #2940. The home folder was not expanded I think.

**Comment by tonyabracadabra** [comment]
> This will help on that #2940. The home folder was not expanded I think.

it works now, but is there any reason why it works in `read_csv` while `scan_csv`  is falling on it? It would be nice if they share the same path reading logic

**Comment by cjermain** [comment]
Thanks for mentioning that. I've updated #2940 to support the `scan_` methods as well.

**Comment by zundertj** [comment]
Closing, as this issue has been fixed on the current version (Polars 0.13.25, Python 3.10, W10). If there are still problems with home directory, feel free to open a new issue.
